index.html

```diff
@@ -1,199 +1,305 @@
+                        <li>Ruby</li>
+                        <li>HMTL</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+            </div>
 
-                    <div>
-                        <div class="square-card">
-                            <div class="language-img">
-                            <img src="images-2/third-icon.svg" alt="skills">
-                            </div>
-                        <h3>skills</h3>
-                        <ul class="my-languages">
-                            <li>Database Management</li>
-                            <li>Version Controle</li>
-                            <li>CLI</li>
-                            <li>WEb Development</li>
-                            <li>API Design</li>
-                        </ul>
-                        </div>
+            <div>
+                <div class="square-card">
+                    <div class="language-img">
+                        <img src="images-2/second-icon.svg" alt="Framewokrs">
                     </div>
+                    <h3>frameworks</h3>
+                    <ul class="my-languages">
+                        <li>React.js</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
 
+            <div>
+                <div class="square-card last-square-card">
+                    <div class="language-img">
+                        <img src="images-2/third-icon.svg" alt="skills">
                     </div>
-                    
-                    
-                </section>
-        </article>
-        <footer>
-            <div class="data">
-            <p class="comment">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-            <form  action="https://formspree.io/f/xqkovozr" method="POST" class="form">
-              <input class="input_name" type="text" placeholder="Full name" name="name" maxlength="30" required/>
-              <input class="input_email" type="email" placeholder="Email address" name="email" required="email"/>
-              <textarea class="input_msg" placeholder="Hey I'd like to connect to..." name="message" rows="8" maxlength="500" required ></textarea>
-              <div class="Get-in-touch-button"><button class="interaction-button" type="submit">Get in touch</button></div>
+                    <h3>skills</h3>
+                    <ul class="my-languages">
+                        <li>Database Management</li>
+                        <li>Version Controle</li>
+                        <li>CLI</li>
+                        <li>WEb Development</li>
+                        <li>API Design</li>
+                    </ul>
+                </div>
+            </div>
+
+        </div>
+
+</div>
+    </section>
```

Comment:
As I already explained, the elements should be centered, and the spacing needs to be updated to match the design. I recommend checking the layout and spacing of your website across different screen sizes: from 375px to 768px for the mobile version, and from 768px up to a minimum of 1440px for the desktop version. 

Please follow the design closely and make the necessary adjustments to ensure that the section is centered and the spacing is consistent across different screen sizes. Taking these steps will result in a more visually pleasing and responsive design. Keep up the good work and pay attention to the details! 😄